File: scripts/auto-swap.py
#!/usr/bin/env python3
"""Delegates to auto-swap-dust.js (Jupiter Swap API v2)."""
import os
import subprocess
import sys

ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
cmd = ["node", os.path.join(ROOT, "scripts", "auto-swap-dust.js"), *sys.argv[1:]]
sys.exit(subprocess.call(cmd, cwd=ROOT))

File: scripts/auto-swap.js
#!/usr/bin/env node
/** @deprecated Use auto-swap-dust.js — kept as alias for Hermes/cron references. */
import { runAutoSwapDust } from "./auto-swap-dust.js";

runAutoSwapDust().catch((err) => {
  console.error(err.message || err);
  process.exit(1);
});